### PR TITLE
Consolidate ray tracing tests

### PR DESCRIPTION
**Pull request by nmnobre**
Hi @GiudGiud,

This _should_ be the last one of its kind, at least for a while 😇.

Cheers,
-Nuno

### CODE REVIEW

File: modules/ray_tracing/test/tests/traceray/edge2/tests [hunk 1]
```diff
@@ -16,6 +16,6 @@
     allow_test_objects = true
     abs_zero = 1e-8
     method = '!dbg'
-    requirement = 'The system shall be able to trace rays within EDGE2 elements.'
+    requirement = 'The system shall be able to trace rays within EDGE elements.'
```
[the remark this answers is not in the document]
Comment (cticenhour) [reply modules/ray_tracing/test/tests/traceray/edge2/tests:19]: I don't think we should allow symlinks for test files for SQA reasons, most importantly. Requirements should and will always be different within every `tests` file. Accomplishing that with a single file symlinked in many places could potentially be tricky and messy. 

If I think further, hypothetical adjustments required to a given test spec due to changes in MOOSE handling of particular elem types would be made more tedious with symlinks.

[the remark this answers is not in the document]
Comment (nmnobre) [reply modules/ray_tracing/test/tests/traceray/edge2/tests:19]: > this trick is going to fail SQA due to duplicate requirement

Is there an automated test for this?

[the remark this answers is not in the document]
Comment (GiudGiud) [reply modules/ray_tracing/test/tests/traceray/edge2/tests:19]: > Is there an automated test for this?

you ll see the doco recipe on CIVET fail once they are done

[the remark this answers is not in the document]
Comment (nmnobre) [reply modules/ray_tracing/test/tests/traceray/edge2/tests:19]: I see no clean way around this I'll be honest.
It seems there's no brace expression support when reading test HIT files, I suppose that's by design?

[the remark this answers is not in the document]
Comment (GiudGiud) [reply modules/ray_tracing/test/tests/traceray/edge2/tests:19]: Yeah I dont see one either. I dont see how to move this forward. 
We have to live with the duplication of the tests file.

**Pull request closed without merging**